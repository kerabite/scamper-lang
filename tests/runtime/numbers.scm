4129
0
-48902
+48902
00142
-089
+098

3.14
.14
0.14
314.

-3.14
-.14
-0.14
-314.

+3.14
+.14
+0.14
+314.

3e2
3.0e2
.3e2
3E2
3.0E2
.3E2

-3e2
-.3e2
-3e-2
-.3e-2