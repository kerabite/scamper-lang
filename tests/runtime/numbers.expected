4129
0
-48902
48902
142
-89
98
3.14
0.14
0.14
314
-3.14
-0.14
-0.14
-314
3.14
0.14
0.14
314
300
300
30
300
300
30
-300
-30
-0.03
-0.003
